csharp/Adapter/Microsoft.Spark.CSharp/Proxy/Ipc/SparkContextIpcProxy.cs

```diff
@@ -361,6 +361,18 @@ public IColumnProxy CreateWindowFunction(string name)
             }
             return jmap;
         }
+
+        public static JvmObjectReference GetJavaHashMap<K, V>(IEnumerable<KeyValuePair<K, V>> enumerable)
```

Comment:
Would it be better to use a dictionary instead of IEnumerable<KVP>? With that, we do not have to check for duplicate keys.